diff --git a/lib/qless/server/views/layout.erb b/lib/qless/server/views/layout.erb
index cb41fb6..9ac8747 100644
--- a/lib/qless/server/views/layout.erb
+++ b/lib/qless/server/views/layout.erb
@@ -12,8 +12,8 @@
     <link href="<%= u '/css/docs.css' %>" rel="stylesheet">
     <link href="<%= u '/css/jquery.noty.css' %>" rel="stylesheet">
     <link href="<%= u '/css/noty_theme_twitter.css' %>" rel="stylesheet">
     <link href="<%= u '/css/style.css' %>" rel="stylesheet">
-    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+    <script src="<%= u '/js/jquery.min.js' %>" type="text/javascript"></script>
 
     <style type="text/css">
     body {
